### DISCUSSION TITLE
(Feature request) prompt user to update the game

### DISCUSSION THREAD

**Discussion by ghost** [aldostools/webMAN-MOD#979] [Ideas]
Basically we prompt the updates from webman database whether it's digital or physical game. If they don't want to update their game simply hit no or press the back button. Hitting yes will automatically download and install game updates. Pretty neat right?

**Comment by ghost** [comment]
No PSN involved with this feature.

**Reply by Darkhost1999** [reply]
PSN was never involved with game updates on the PS3. Only Internet.
So this feature request is the basic functionality of the PS3.
Webman mods list uses the same exact URLs from Sony's API with no deviation including DRM free updates for the 11 or so games that feature that.

**Reply by ghost** [reply]
ok

**Comment by aldostools** [comment]
Once the game is mounted, there is a context menu pressing TRIANGLE that lets update the game.

The webMAN Games menu does not have a context menu for URL links.
